types/rfc4648/index.d.ts

```diff
@@ -0,0 +1,63 @@
+// Type definitions for rfc4648 1.2
+// Project: https://github.com/swansontec/rfc4648
+// Definitions by: Thibault Meunier <https://github.com/thibmeu>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+type parseBase = <T = Uint8Array>(
+    data: string,
+    options?: {
+        out?: { new(): T };
+        loose?: boolean;
+    },
+) => T;
+
+type stringifyBase = (data: object, options?: { pad?: boolean }) => string;
+
+export namespace base64 {
```

Comment:
If you want, you could consider adding a type
```
interface DataFormatBase {
  parse: parseBase;
  stringify: stringifyBase
}

export const base64: DataFormatBase;
```

Reply:
That's what I did before, but compiler complains since I'm not exporting `DataFormatBase` (and I don't want to).